lesson_01/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"github.com/kyokomi/emoji/v2"
+)
+
+func main() {
+
+	// :face: not support
```

Comment:
What is this comment about?

Reply:
> What is this comment about?

smile with type ":face:" doesn't contains in emoji pack